Jenkinsfile

```diff
@@ -33,7 +33,7 @@ properties([
            defaultValue: '1',
            description: 'How many times to repeat each test.'),
     string(name: 'ROBOTEST_VERSION',
-           defaultValue: 'stable-gce',
+           defaultValue: 'uid-gid',
```

Comment:
I'm assuming this is going to change back to stable tag at some point?

Reply:
I probably won't change it back to a "stable tag", because the named feature tag pattern we're currently using is atypical.  If you're concerned about stability, both stable-gce and uid-gid are releases off robotest's master branch, and I believe uid-gid is a direct descendant of stable-gce (though I haven't proved it yet).

I'd like to return robotest to semver release tagging/naming but doing so was not higher priority than this work. I chose not to fix semver for this particular task because our current semver is 1.0.217 or something silly -- with several backwards incompatible changes (e.g. not attempting to maintain AWS and Azure support). I'll cut a 2.0 when I make a bit of priority to revisit our Robotest release process.

Reply:
Yeah, for sure, I just meant to use a some kind of "proper" tag eventually because "uid-gid" is sort of like "dima" :)

Reply:
I've filed https://github.com/gravitational/robotest/issues/200 to track this.